.idea/libraries/GOPATH__hologram_.xml

```diff
@@ -0,0 +1,15 @@
+<component name="libraryTable">
+  <library name="GOPATH &lt;hologram&gt;">
+    <CLASSES>
+      <root url="file://$USER_HOME$/src/go/src/golang.org" />
+      <root url="file://$USER_HOME$/src/go/src/github.com" />
+    </CLASSES>
+    <SOURCES>
+      <root url="file://$USER_HOME$/src/go/src/golang.org" />
```

Comment:
just making me crazy - using jetbrains and i keep accidentally committing this crap... hopefully removed in next commit. thanks!